Mathlib/Data/Fintype/Lattice.lean

```diff
@@ -67,3 +67,19 @@ theorem Finite.exists_min [Finite α] [Nonempty α] [LinearOrder β] (f : α →
   cases nonempty_fintype α
   simpa using exists_min_image univ f univ_nonempty
 #align finite.exists_min Finite.exists_min
+
+theorem iSup_finsetUniv [CompleteLattice α] [Fintype β] {f : β → α} :
```

Comment:
Doesn't this (and it's `iInf` counterpart) appear above as `Finset.sup_univ_eq_iSup`?

Reply:
it needs an additional non-simp/non-defeq conversion:
```lean
theorem iSup_finsetUniv [CompleteLattice α] [Fintype β] {f : β → α} :
    ⨆ x ∈ Finset.univ, f x = ⨆ x, f x := by
  rw [← Finset.sup_eq_iSup, Finset.sup_univ_eq_iSup]
  -- or: rw [← Finset.iSup_coe, Finset.coe_univ, iSup_univ]
```

I guess this one depends on whether it's standard to copy lemmas through equivalent constructs, e.g. `Finset.univ_sigma_univ` vs `Set.univ_sigma_univ`, `Finset.sup_union` vs `Finset.iSup_union`, [etc](https://github.com/leanprover-community/mathlib4/blob/8666bd82efec40b8b3a5abca02dc9b24bbdf2652/Mathlib/Data/Finset/Lattice.lean#L2021-L2083).